scss/_custom-forms.scss

```diff
@@ -87,6 +84,10 @@
     background-position: center center;
     background-size: $custom-control-indicator-bg-size;
   }
+
+  &:not(:empty)::before {
+    margin-right: $custom-control-gutter;
```

Comment:
`.custom-control-label` will never be empty because of the before and after pseudo-elements. It will always have `margin-right`.